scripts/pipeline-launch/launch.sh

```diff
@@ -1678,7 +1678,15 @@ pipe_log SUCCESS "Environment initialization finished" "InitializeEnvironment"
 
 echo "Command text:"
 echo "${SCRIPT}"
-bash -c "${SCRIPT}"
+
+if [ ! -z "${CP_EXEC_TIMEOUT}" ] && [ "${CP_EXEC_TIMEOUT}" -gt 0 ];
+then
+  timeout ${CP_EXEC_TIMEOUT}m bash -c "${SCRIPT}"
+  echo "Timeout was elapsed"
```

Comment:
I think that we need to check whether timeout was actually exceeded (possibly using exit code). Also keep in memory that we need to save `CP_EXEC_RESULT` to exit process with correct exit code.